src/maintainer/knowledge_base.rst

```diff
@@ -748,3 +748,29 @@ Following implies that ``python`` is a runtime dependency and a Python matrix fo
 
 You need to rerender the feedstock after this change.
 
+Requiring newer macOS SDKs
+==========================
+
+conda-forge uses macOS SDK 10.9 to build software so that they can be deployed to
+all macOS versions newer than 10.9. Sometimes, some packages require a newer SDK
+to build with. While the default version 10.9 can be overridden using the following
+changes to the recipe, it should be done as a last resort.
```

Comment:
Should we add something like, "Please consult with core if this is something you think you need."?

Reply:
Sure. Please go ahead and push to this branch.